resources/vue/components/NotificationBell.vue

```diff
@@ -0,0 +1,110 @@
+<script setup>
+import ModalComponent from "./ModalComponent.vue";
+import {onMounted, reactive, ref} from "vue";
+import NotificationList from "./NotificationList.vue";
+
+defineProps({
+    label: {
+        type: String,
+        default: "aria-label placeholder",
+    },
+    link: {
+        type: Boolean,
+        default: false
+    },
+    i18nEmpty: {
+        type: String
+    },
+    i18nTitle: {
+        type: String
+    }
+});
+
+onMounted(() => {
+    fetchCount();
+    setInterval(fetchCount, 30000);
```

Comment:
Intervals should be properly destroyed, see [onUnmounted()](https://vuejs.org/api/composition-api-lifecycle.html#onunmounted)